### ISSUE TITLE
Package request: mesa-amber

### ISSUE THREAD

**Issue by charwhee** [void-linux/void-packages#48519]
### Package name

mesa-amber

### Package homepage

https://gitlab.freedesktop.org/mesa/mesa/-/blob/amber/docs/relnotes/21.3.9.rst

### Description

Legacy intel video drivers were removed from the mesa package and moved to mesa-amber.

"After Mesa 21.3, all non-Gallium DRI drivers were removed from the Mesa source-tree. These drivers are still being maintained to some degree, but only on the amber branch, and only for critical fixes.

These drivers include:
    Radeon
    r200
    i915
    i965
    Nouveau (the DRI driver for NV04-NV20)"

Reference Documentation:
https://docs.mesa3d.org/amber.html
https://archlinux.org/packages/?q=mesa-amber

### Does the requested package meet the package requirements?

System, Compiled, Required

### Is the requested package released?

Yes

**Comment by MIvanchev** [comment]
What is the point of this package. Most (all?) of the devices are still supported by Mesa.

**Comment by charwhee** [comment]
Yes, it's for older computers, and Void is one of the only distributions that still support 32-bit architectures, so it makes sense to support older video drivers. If it's not something you want to prioritize, totally understand (I can see all the requests you get). I'll take a crack at it - I suspect it will be identical to mesa except for 'amber' in the name. Cheers!

**Comment by MIvanchev** [comment]
>Yes, it's for older computers

What I wanted to say is that the Gallium drivers support these devices as well so older computers should still be good.

**Comment by 20NE** [comment]
Even arch linux with mesa-amber package

**Comment by classabbyamp** [comment]
I'd be happy to accept such a package if someone with the hardware to use and test it were to maintain it.

Also, "[other distro] has [package]" is not valid justification for adding a package

**Comment by 20NE** [comment]
not [other distro], I said "arch linux"

**Comment by classabbyamp** [comment]
arch is not void, so it is another distro

**Comment by 20NE** [comment]
One way Arch not that bloat, BTW

**Comment by Lana-chan** [comment]
+1 for the request, have a ThinkPad T42 with Radeon 7500 (r100 driver) that would benefit from this

**Comment by DCFUKSURMOM** [comment]
+1. Machine is a Toshiba Satellite M115-S1061. ATI Mobility Radeon Xpress 200M, not sure exactly what mesa driver it needs, but its not enabled in the regular mesa package. Had HW accel in either Adelie or Arch32, so I know its possible, but Adelie kinda sucks and Arch32 is sadly broken in general

**Comment by DCFUKSURMOM** [comment]
> +1. Machine is a Toshiba Satellite M115-S1061. ATI Mobility Radeon Xpress 200M, not sure exactly what mesa driver it needs, but its not enabled in the regular mesa package. Had HW accel in either Adelie or Arch32, so I know its possible, but Adelie kinda sucks and Arch32 is sadly broken in general

Ok so manually editing the Xorg config to enable DRI3 for this card got acceleration working, not sure if that works for any other cards mentioned

**Comment by DCFUKSURMOM** [comment]
While it was not needed in my case specifically, hiding comments that give legitimate reasons why this is needed is a dick move.

**Comment by classabbyamp** [comment]
> I'd be happy to accept such a package if someone with the hardware to use and test it were to maintain it.

I've already said I'd accept the package if someone submitted a pull request, so additional +1s and comments of that nature aren't necessary
